llm/tests/bench_serving.py

```
@@ -0,0 +1,198 @@
+        "The meaning of life is",
+        "Python is a programming language that",
+        "In the year 2050,",
+        "The fastest animal on Earth is",
+    ]
+
+    tasks = []
+    for i in range(args.num_requests):
+        prompt = prompts[i % len(prompts)]
+        tasks.append(prompt)
+
+    print(f"=== PyPTO Serving Benchmark ===")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)